pykokkos/interface/views.py

```diff
@@ -704,6 +706,53 @@ def from_cupy(array) -> ViewType:
 
     return from_numpy(np_array, memory_space, layout)
 
+def is_array(array) -> bool:
+    """
+    Check if an object conforms to enough numpy array standards to be treated as an array
+    Using the function from_cupy() as a standard for bare minimum needed to
+    to convert any array into a numpy array
+
+    :param array: the array of unknown type
+    :returns: a true/false if object is an array
+    """
+    
+    required_attr = ["dtype", "data", "shape", "flags"]
+    test_attr = dir(array)
+
+    if(not set(required_attr).issubset(set(test_attr))):
+      return False
+    
+    for d in required_attr:
```

Comment:
this loop is confusing: if we enter we either return True or False, why do we have a loop? `return True` outside `for`

Reply:
an attribute can be callable (a class function) or not (class variable). If the attribute happens to have the same name but is callable (as aoopsed to plain data)... I think the conversion would crash and burn.

Reply:
My question is about the return statement inside the loop body (not outside)